### DISCUSSION TITLE
increase precision of contrast_samples

### DISCUSSION THREAD

**Discussion by dorcatty** [kurokida/jsQuestPlus#9] [General]
Dear jsQuestPlus auther, 

I have one simple question about how to increase input precision, i went through all demos, it seems the majority sample codes were using:
    const contrast_samples = jsQuestPlus.linspace(-40, 0) // [-40, -39, -38, ..., -1, 0]
which generates an array of input value from -40~0, step 1. 
and only one code using double precision like :
    const contrast_samples = jsQuestPlus.linspace(0, 1, 100)
which generates an arry of input value from 0~1, step 1/100

my question is how to generate a input range of 0~1, step 1/1000
i tried with jsQuestPlus.linspace(0, 1, 1000)
but this will slowdown significantly the implementation of code, my browser goes into BUSY forever. 

is there any solution to generate more precise input values ? thanks

**Comment by kurokida** [comment]
Thank you for trying to use jsQuestPlus.

Have you already checked [this wiki page](https://github.com/kurokida/jsQuestPlus/wiki/Customization-of-Psychometric-function)?

An input range of 0~1, step 1/1000 seems to be too fine. I think this setting imposes a lot of trials on the participants. I recommend measuring at a much coarser stimulus intensity.

Best regards,
Daiichiro
